core/src/main/java/hudson/model/Executor.java

```diff
@@ -37,6 +37,7 @@
 import jenkins.model.CauseOfInterruption.UserInterruption;
 import jenkins.model.InterruptedBuildAction;
 import jenkins.model.Jenkins;
+import net.jcip.annotations.GuardedBy;
```

Comment:
I maintain that Jenkins is using the wrong ones as there is no JSR defining the javax ones and thus they are subject to change ;-) (but yeah it's unlikely they will change and thus I'll fix)